packages/lucide-svelte/scripts/appendLicense.mjs

```diff
@@ -1,13 +1,8 @@
-import { getCurrentDirPath } from '../../../scripts/helpers.mjs';
```

Comment:
I think we can better rename this to `appendBlockComments.mjs`. I've also added other block comments for the icon components.